Pipfile

```diff
@@ -9,7 +9,9 @@ pylint = "*"
 
 [packages]
 python-telegram-bot = "*"
+psycopg2 = "*"
+setuptools = "*"
```

Comment:
Why do we need this?

Reply:
psycopg2 is the postgres adaptor for python. setuptools is redundant.